src/System.ComponentModel.TypeConverter/src/System/Drawing/FontConverter.cs

```diff
@@ -0,0 +1,399 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.ComponentModel;
+using System.ComponentModel.Design.Serialization;
+using System.Drawing.Text;
+using System.Globalization;
+using System.Reflection;
+using System.Text;
+
+namespace System.Drawing
+{
+    public class FontConverter : TypeConverter
+    {
+        public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
+        {
+            return sourceType == typeof(string) ? true : base.CanConvertFrom(context, sourceType);
+        }
+
+        public override bool CanConvertTo(ITypeDescriptorContext context, Type destinationType)
+        {
+            if (destinationType == typeof(string))
+            {
+                return true;
+            }
+            if (destinationType == typeof(InstanceDescriptor))
+            {
+                return true;
+            }
+            return base.CanConvertTo(context, destinationType);
+        }
+
+        public override object ConvertTo(ITypeDescriptorContext context, CultureInfo culture, object value, Type destinationType)
+        {
+            if ((destinationType == typeof(string)) && (value is Font))
+            {
+                Font font = (Font)value;
+                StringBuilder sb = new StringBuilder();
+                sb.Append(font.Name).Append(culture.TextInfo.ListSeparator[0] + " ");
+                sb.Append(font.Size);
+
+                switch (font.Unit)
+                {
+                    // MS throws ArgumentException, if unit is set
+                    // to GraphicsUnit.Display
+                    // Don't know what to append for GraphicsUnit.Display
+                    case GraphicsUnit.Display:
+                        sb.Append("display");
```

Comment:
Instead of using `StringBuilder` this could be done with string.Create span api: https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/System/String.cs#L346

Reply:
I do not understand this one. How can I _insert_ something into `Span` when I need to add it by smaller chunks?

Reply:
You could do something like this: https://github.com/dotnet/corefx/pull/30528/files

But I don't think is worth the effort as this is not a hot path and the code is going to get somehow larger. 

However you could use ValueStringBuilder instead. 

https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/Text/ValueStringBuilder.cs